Tasks/AzureCloudPowerShellDeploymentV1/Utility.ps1

```diff
@@ -71,7 +87,7 @@ function Get-DiagnosticsExtensions($storageAccount, $extensionsPath, $storageAcc
         Write-Host (Get-VstsLocString -Key "Applyinganyconfigureddiagnosticsextensions")
 
         Write-Verbose "Getting the primary AzureStorageKey..."
-        $primaryStorageKey = (Get-AzureStorageKey -StorageAccountName "$storageAccount").Primary
+        $primaryStorageKey = Get-StorageAccountPrimaryKey $StorageAccount $useArmStorage.IsPresent
```

Comment:
Should this be Get-AzureStoragePrimaryKey?

Reply:
Fixed